decentraland/screens/BuyLandScreen.js

```diff
@@ -76,7 +76,10 @@ export class BuyLandScreen extends React.Component {
       // TODO: This function should be called inside of the eventListener
       // that catches the safeExecuteOrder successful event.
       await action.waitForNonceToUpdate();
-      showInfo(`Asset bought successfully.`);
+
+      if (type == ESTATE) showInfo(`Estate bought successfully.`);
+      else if (type == PARCEL) showInfo(`Parcel bought successfully.`);
```

Comment:
Should we do an else to log an error? I like to handle all possible cases whenever possible

Reply:
OK, done.

Reply:
Thanks! Resolving